### DISCUSSION TITLE
Import Definitions for migration purposes

### DISCUSSION THREAD

**Discussion by andrescolodrero** [rabbitmq/cluster-operator#1384] [General]
Hi,
i generated a new cluster and now im planning a migration with a blue-green deployment.
Based on rabbitmq, first stat with export/import of definitions and  them configuring federated queues.

Definitions can be a big file, how to import with the operator?(the instrucctions i see are with rabbitmqctl command)

**Comment by andrescolodrero** [comment]
close the issue, perfectlt explained here: https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/import-definitions

**Reply by mkuratczyk** [reply]
Just for completeness, there's also an option to import over HTTP:
https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/import-https-url-definition
